### PR TITLE
Gatsby 2 (for real)

### LINKED ISSUE #361: Gatsby v.2
Hey all 👋! Would you take a PR to upgrade to Gatsby v.2, or are we waiting for the official release?

cc: @lachlanjc

### LINKED ISSUE #463: Migrate admin tools to separate codebase

### PR DESCRIPTION
**Pull request by merelinguist**
Ready for review, should all be complete and working. Would love to get merged before it goes too stale.

Closes #361 
Closes #463

### CODE REVIEW

File: src/components/workshops/CarouselSubmissionForm.js [hunk 1]
```diff
@@ -62,7 +62,7 @@ class CarouselSubmissionForm extends Component {
         headers: { 'Content-Type': 'application/json' }
       })
       // For now, just refresh the page. Needs a real Submssion Complete page eventually.
-      .then(() => location.reload())
+      .then(() => window.locaction.reload())
```
Comment (lachlanjc) [inline src/components/workshops/CarouselSubmissionForm.js]: Typo

**Review by lachlanjc — APPROVED** [review]
Wow, amazing work Dylan. Thank you so much, & for going above & beyond with Hooks & all. Will take a closer look through every diff before merge but seriously, thank you 😇

**Comment by lachlanjc** [comment]
- [x] Fix Netlify build settings (upgrading Yarn + Node)
- [x] Lachlan should test the whole app
- [x] Manually import `graphql` like `import { graphql } from 'gatsby'` everywhere
- [x] Rebase after workshop-drafting addition

**Comment by merelinguist** [comment]
- [x] Fix workshop search
- [x] https://hackclub.com/workshops/personal_website/ the loading circle

**Comment by merelinguist** [comment]
### Check all pages
- [x] apply
  - [x] club
  - [x] index
  - [x] leader
- [x] 404
- [x] bank
- [x] banner
- [x] camp
- [x] challenge
- [x] cloud9_setup
- [x] conduct
- [ ] confirm_invite
- [x] donate
- [x] hackers
- [x] index
- [x] night
- [x] philosophy
- [x] redeem_tech_domain
- [x] santa
- [x] slack_invite
- [x] start
- [x] team
- [x] updates
- [x] workshops
- [x] workshop pages

**Pull request merged**